docusaurus/docs/learn/quickstarts/network/help/quickstart-walkthrough.md

```diff
@@ -0,0 +1,314 @@
+---
+title: Quickstart Walkthrough
+id: quickstart-walkthrough
+---
+The following will walk you through the manual process of creating an overlay network with your own PKI. These steps are the same steps that take place behind the scenes when running the OpenZiti Quickstart’s `expressInstall` function.
+
+## General Environment Setup
+
+### Declare Variables
+
+Some global variables used throughout the overlay setup.
+
+- `ZITI_HOME` is the directory where your network files will be stored, the directory will be created if it does not exist already.
+- `ZITI_BIN_DIR` is the directory where the ziti binary will be downloaded and extracted
+- `ZITI_PWD` is the password that will be used when logging into the controller as the default user (admin).
+- `ZITI_HOSTNAME` is used throughout the setup for naming files, network elements, etc.
+
+```
+export ZITI_HOME="${HOME}/.ziti"
+export ZITI_BIN_DIR="${ZITI_HOME}/ziti_bin"
+export ZITI_PWD=admin2
+export ZITI_HOSTNAME=$(hostname -s)
+```
+
+### Create Environment
+
+Create a directory where you will store your network files.
+
+```
+mkdir -p "${ZITI_HOME}"
+```
+
+### Obtain Ziti Binary
+
+The ziti binary is required to setup the network. The exact URL will differ depending on your operating system and architecture. Visit the [releases](https://github.com/openziti/ziti/releases) page to get the appropriate URL.
+
+```
+mkdir -p "${ZITI_BIN_DIR}"
+curl -L <release-url> -o "${ZITI_BIN_DIR}/ziti-bin.gz"
+tar -xf "${ZITI_BIN_DIR}/ziti-bin.gz" --directory "${ZITI_BIN_DIR}"
+chmod +x "${ZITI_BIN_DIR}/ziti"
+```
+
+## Create PKI
+
+You may use an existing PKI if you prefer, however, as part of the `expressInstall` a PKI is generated for you. The following will run through the process of generating your own PKI.
+
+### Setup
+
+Set some initial environment variables for setting up the PKI. The `CA_NAME` values can be any name of your choosing.
+
+- `ZITI_PKI` is the directory where your PKI files will be stored
+- `ZITI_ROOT_CA_NAME` is the name of the root CA.
+- `ZITI_EXTERNAL_CA_INTERMEDIATE_NAME` is the name of an intermediate CA.
+- `ZITI_CTRL_CA_NAME` is the name of the control plane CA.
+- `ZITI_EDGE_CA_NAME` is the name of the HTTP API CA
+- `ZITI_SIGN_CA_NAME` is the name of the signer CA used to sign identities created for the network.
+
+```
+export ZITI_PKI="${ZITI_HOME}/pki"
+export ZITI_ROOT_CA_NAME="my.root.ca"
+export ZITI_EXTERNAL_CA_INTERMEDIATE_NAME="intermediate.from.external.ca"
+export ZITI_CTRL_CA_NAME="${ZITI_HOSTNAME}-network-components"
+export ZITI_EDGE_CA_NAME="${ZITI_HOSTNAME}-edge"
+export ZITI_SIGN_CA_NAME="${ZITI_HOSTNAME}-identities"
+```
+
+### Creating the Certificate Authorities
+
+The following creates all of the CAs and files using the environment variables set up previously.
```

Comment:
You should explain that we're making a complex PKI here with a root ca, three intermediate ca's one for each of the controller's config sections and even adding an extra intermediate on the signing cert just to demonstrate that arbitrary cert chain lengths are fine